superset-frontend/src/components/DatabaseSelector/DatabaseSelector.test.tsx

```diff
@@ -76,6 +76,7 @@ beforeEach(() => {
             allows_cost_estimate: 'Allows Cost Estimate',
             allows_subquery: 'Allows Subquery',
             allows_virtual_table_explore: 'Allows Virtual Table Explore',
+            disable_preview_data: 'Disables SQL Lab Data Preview',
```

Comment:
nit: would `disable_data_preview` be a more appropriate name?

Reply:
Implemented in most recent commit